### PR TITLE
RAD-502: CRDS Keywords Not Required 

### LINKED ISSUE #502: CRDS Keywords Not Required
The CRDS keywords should be required:  https://github.com/spacetelescope/rad/blob/a82c27425bbd7fbeec2c191c3ac13cfec9916cc3/src/rad/resources/schemas/ref_file-1.0.0.yaml#L9

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RAD-1234](https://jira.stsci.edu/browse/RAD-nnnn)

Closes #502

This PR adds CRDS and reference steps required lists.

## Tasks
- [ ] Update or add relevant `rad` tests.
- [ ] Update relevant docstrings and / or `docs/` page.
- [x] Does this PR change any schema files?
  - [ ] Schema changes were discussed at RAD Review Board meeting.
- [x] Does this PR change any API used downstream? (If not, label with `no-changelog-entry-needed`.)
  - [ ] Write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types).
  - [ ] Start a `romancal` regression test (https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/rad@<branch>"`).
  - [x] Update relevant `roman_datamodels` utilities and tests.

<details><summary>News fragment change types:</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

**Comment by PaulHuwe** [comment]
> This looks like a duplicate of #505

This is the part that is needed for release, and the #505 stuff is for after the release.

File: src/rad/resources/schemas/ref_file-1.0.0.yaml [hunk 1]
```diff
@@ -171,4 +172,5 @@ properties:
       datatype: nvarchar(120)
       destination: [ScienceRefData.r_refpix, GuideWindow.r_refpix, WFICommon.r_refpix]
 flowStyle: block
+required: [crds, dark, distortion, mask, flat, gain, readnoise, linearity, photom, area, saturation, refpix]
```
[the remark this answers is not in the document]
Comment (PaulHuwe) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:175]: Not offhand - given that the other change will break files, I didn't look

[the remark this answers is not in the document]
Comment (schlafly) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:175]: Okay.  Let's wait until after standup and I ping Melissa about any other issues she may have identified.  But then let's limit this to the minimum change to fix the problem (just making the two crds keywords required, I think?), and then merge unless there are more problems identified.

I think the other required changes are good but but aren't necessary to get things to work.

**Comment by schlafly** [comment]
Yes, it will.

File: src/rad/resources/schemas/ref_file-1.0.0.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (PaulHuwe) [reply src/rad/resources/schemas/ref_file-1.0.0.yaml:175]: I think so - we know that file swill need to be remade with these changes.

**Pull request merged**